src/main/java/dev/tycho/stonks/command/company/PayCommandSub.java

```diff
@@ -0,0 +1,101 @@
+package dev.tycho.stonks.command.company;
+
+import dev.tycho.stonks.command.base.CommandSub;
+import dev.tycho.stonks.gui.AccountSelectorGui;
+import dev.tycho.stonks.gui.CompanySelectorGui;
+import dev.tycho.stonks.gui.ConfirmationGui;
+import dev.tycho.stonks.managers.DatabaseHelper;
+import dev.tycho.stonks.model.Company;
+import dev.tycho.stonks.model.Member;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+
+import static dev.tycho.stonks.model.Role.CEO;
+
+public class PayCommandSub extends CommandSub {
+
+  private static final List<String> AMOUNTS = Arrays.asList(
+      "1",
+      "10",
+      "1000",
+      "10000");
+
+  @Override
+  public List<String> onTabComplete(CommandSender sender, String alias, String[] args) {
+    if (args.length == 2) {
+      return copyPartialMatches(args[1], AMOUNTS);
+    }
+    return null;
+  }
+
+  @Override
+  public void onCommand(Player player, String alias, String[] args) {
+    if (args.length == 1) {
+      sendMessage(player, "Correct usage: " + ChatColor.YELLOW + "/" + alias + " pay <amount> [<message>]");
+      return;
+    }
+
+    if (!Pattern.matches("([0-9]*)\\.?([0-9]*)?", args[1])) {
```

Comment:
This could be moved to a method, like "validateDouble" in the super class, there are a few places that need double validation